applications/matter_bridge/src/bridged_device_factory.h

```diff
@@ -35,6 +35,12 @@
 #endif
 #endif
 
+#ifdef CONFIG_BRIDGED_DEVICE_BT
+#if defined(CONFIG_BRIDGE_HUMIDITY_SENSOR_BRIDGED_DEVICE) || defined(CONFIG_BRIDGE_TEMPERATURE_SENSOR_BRIDGED_DEVICE)
```

Comment:
```suggestion
#if defined(CONFIG_BRIDGE_HUMIDITY_SENSOR_BRIDGED_DEVICE) && defined(CONFIG_BRIDGE_TEMPERATURE_SENSOR_BRIDGED_DEVICE)
```
Can they be separated in your scenario of environmental device type?

Reply:
We discussed it actually makes no sense to support them separately.